### PR TITLE
Manual test with simple image

### LINKED ISSUE #15185: PoC for image attributes
Prepare a PoC how could show using the new image attributes after the recent changes (#14146).

* no usage of `style` attributes in the data.
* data downcast saves in attributes `width` and `height` a resized image values.
* on upcast, `width` and `height` are set as `resized*`.
* add attributes with intrinsic size `width` and `height`.

### PR DESCRIPTION
**Pull request by niegowski**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Internal (image): Added manual test with overridden image conversion to use only size attributes for images. Closes #15185.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: packages/ckeditor5-image/tests/manual/simple-image.ts [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+			},
+			converterPriority: 'high'
+		} );
+
+	editor.editing.view.addObserver( ImageLoadObserver );
+
+	const imageUtils: ImageUtils = editor.plugins.get( 'ImageUtils' );
+
+	// Waiting for any new images loaded, so we can set their natural width and height.
+	editor.editing.view.document.on( 'imageLoaded', ( evt, domEvent ) => {
```
Comment (arkflpc) [inline packages/ckeditor5-image/tests/manual/simple-image.ts:256]: You used event type in line 33. Why don't do that here.

File: packages/ckeditor5-image/tests/manual/simple-image.ts [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+				wasFixed = true;
+			}
+		}
+
+		return wasFixed;
+	} );
+}
+
+ClassicEditor
+	.create( document.getElementById( 'editor' ), {
```
Comment (arkflpc) [inline packages/ckeditor5-image/tests/manual/simple-image.ts:331]: It does not compile for me. `getElementById` can return `null`.

File: packages/ckeditor5-image/tests/manual/simple-image.ts [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+		const images = [];
+		let wasFixed = false;
+
+		for ( const change of changes ) {
+			if ( change.type == 'attribute' ) {
+				if ( ![ 'width', 'height', 'resizedWidth', 'resizedHeight' ].includes( change.attributeKey ) ) {
+					continue;
+				}
+
+				const item = change.range.start.nodeAfter;
```
Comment (arkflpc) [inline packages/ckeditor5-image/tests/manual/simple-image.ts:292]: The lines below do not compile for me. `nodeAfter` can return null.

File: packages/ckeditor5-image/tests/manual/simple-image.ts [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+
+				for ( const item of items ) {
+					if ( item.is( 'element', 'imageBlock' ) || item.is( 'element', 'imageInline' ) ) {
+						images.push( item );
+					}
+				}
+			}
+		}
+
+		for ( const item of images ) {
```
Comment (arkflpc) [inline packages/ckeditor5-image/tests/manual/simple-image.ts:308]: In my IDE images are inferred as `never[]`, so the lines below do not compile.

**Review by arkflpc — CHANGES_REQUESTED** [review]

**Review by arkflpc — APPROVED** [review]

**Review by filipsobol — APPROVED** [review]

**Pull request merged**